lib/i18next2js.js

```diff
@@ -36,6 +36,7 @@ const parseGettext = (locale, data, options = {}) => {
     },
     translations: {}
   }
+  const plurals = getPlurals(options)
```

Comment:
maybe this or the other variable (on line 15) needs to be renamed?

Reply:
sorry, is this because of a naming clash or because of the naming being off? ^^;;
They are both local variables so I don' t think there's a clash... what would you rename this to?

Reply:
The plurals variable is used twice in that function for different purposes.
So I would rename the second one to pluralsInfo or similar.

Reply:
those are two separate functions, `getPluralArray` and `parseGettext` :P

Reply:
if the variable were declared twice, the code would not run

Reply:
ohh, sorry didn't notice it on my smartphone 🤦🏻‍♂️